#goal definition
sensor_msgs/Image input_img
float64 map_resolution
float64 Map_Origin_x
float64 Map_Origin_y
int32[] room_number
int32[] room_min_x
int32[] room_max_x
int32[] room_min_y
int32[] room_max_y
int32[] room_center_x
int32[] room_center_y
---
#result definition
sensor_msgs/Image output_img
---
#feedback definition